web/assets/images/download.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 84 65" style="enable-background:new 0 0 84 65;" xml:space="preserve">
+<metadata><?xpacket begin="﻿" id="W5M0MpCehiHzreSzNTczkc9d"?>
+<x:xmpmeta xmlns:x="adobe:ns:meta/" x:xmptk="Adobe XMP Core 5.6-c111 79.158366, 2015/09/25-01:12:00        ">
+   <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+      <rdf:Description rdf:about=""/>
+   </rdf:RDF>
+</x:xmpmeta>
+
+<?xpacket end="w"?></metadata>
+<style type="text/css">
+	.st0{fill:#514744;}
+	.st1{fill:#FCB812;}
+	.st2{fill:#F15A22;}
+</style>
+<g>
+	<g>
+		<g>
+			<path class="st0" d="M70.3,63H13.7c-1.8,0-3.3-1.6-3.3-3.4V30.3c0-0.1,0-0.1,0-0.2c0-0.3,0.2-1.1,0.5-1.6l9.8-15.9
+				c0.6-1,1.7-1.7,2.8-1.7H35c0.7,0,1.2,0.5,1.2,1.2s-0.6,1.3-1.3,1.3H23.4c-0.3,0-0.6,0.2-0.8,0.5l-9.7,15.9c0,0.1-0.1,0.4-0.1,0.6
+				v29.2c0,0.6,0.4,1.1,0.9,1.1h56.6c0.5,0,0.9-0.5,0.9-1.1V30.4c0-0.3-0.1-0.6-0.2-0.7L61.4,14c-0.1-0.3-0.4-0.5-0.8-0.5H49
+				c-0.7,0-1.2-0.5-1.2-1.2s0.5-1.2,1.2-1.2h11.6c1.2,0,2.2,0.6,2.8,1.7l9.7,15.8c0.3,0.5,0.4,1.3,0.5,1.6c0,0.1,0,0.1,0,0.2v29.3
+				C73.6,61.4,72.2,63,70.3,63z"/>
+		</g>
+		<g>
+			<path class="st1" d="M53.7,32.3c-0.1,0.1-0.3,0.4-0.4,0.9l-1,4c-0.5,1.9-2.3,3.5-4.4,3.5H36.1c-2,0-3.9-1.5-4.4-3.5l-1-4
+				c-0.1-0.4-0.3-0.7-0.4-0.9H16.1l0,0v23.1c0,1.1,0.8,1.8,1.8,1.8h48.2c1,0,1.8-0.9,1.8-1.8V32.3H53.7z"/>
+		</g>
+		<path class="st2" d="M53.6,30.3h14.3l-8.1-13.8c-0.2-0.4-0.7-0.7-1.2-0.7h-9l-6,8l-4.9-0.9L34,15.7h-8.8c-0.5,0-0.9,0.3-1.2,0.7
+			l-7.9,13.9h14.3c1.1,0,1.8,1.2,2.1,2.2l1,4c0.3,1.2,1.4,2,2.5,2h11.8c1.1,0,2.2-0.9,2.5-2l1-4C51.7,31.5,52.4,30.3,53.6,30.3z"/>
+	</g>
+	<path class="st0" d="M52.2,15.9c-1-1-3.5-0.8-4.5,0.2l-4.2,4.2V4c0-1.4-0.7-2-2-2c-1.4,0-2,0.7-2,2v16.2l-4-4.4c-1-1-2.6-1-3.6,0
+		s-1,2.6,0,3.6l8.1,8c0.5,0.7,1.3,1.2,2.1,1.2s1.7-0.5,2.1-1.2l8.1-8C53.2,18.5,53.2,16.9,52.2,15.9z"/>
+</g>
+</svg>
```

Comment:
I wanted to do it during https://jira.ez.no/browse/COM-19900
But let's do that now.